src/components/header/Header.tsx

```diff
@@ -0,0 +1,27 @@
+
+import React  from 'react'
+
+import { GrFilter } from 'react-icons/gr'
+import { TbArrowsSort } from 'react-icons/tb'
+import { useLocation } from 'react-router-dom'
+
+import './Header.css'
+
+const Header = () => {
+  const location = useLocation()
+  const showSearchs = location.pathname === '/'
```

Comment:
typo

Reply:
I changed name for showSearch and location consts.